blueprint/designer.html

```diff
@@ -2017,40 +2031,40 @@
     
     console.log(mygroup);
```

Comment:
Please remove all the console.log(). I know these might be some of my console.log(), but we should make sure that the future production code will not have console.logs.